protostar/commands/upgrade/upgrade_command.py

```diff
@@ -93,12 +94,20 @@ def upgrade(self):
             self._pull_tarball()
             self._install_new_version()
             self.cleanup()
-        # pylint: disable=broad-except
-        except (Exception, KeyboardInterrupt, SystemExit) as err:
+        except KeyboardInterrupt:
+            self._rollback()
+            self.cleanup()
```

Comment:
Why not re-raise?

Reply:
There's no point of printing details of this exception. It's an interrupt triggered by a user, not an error.

Reply:
Maybe then it's worth to add a log about interrupted process
`Interrupting...`
cleanup
`Upgrade interupted`